system/mavros2/README.md

```diff
@@ -0,0 +1,147 @@
+# `starling-mavros2` Container
+
+This container is the native ROS2 MAVROS container. Most of the below still holds.
+
+## Contents
+[TOC]
+
+## Launch Process
+
+The initial process of starting the container goes through `ros_entrypoint.sh`. This file sets up both ROS versions and
+runs a setup script which configures the environment to tell MAVROS how to talk to the drone and where to publish its
+topics. The setup script works in a couple of different ways depending on where the container is running.
+
+If the container is running on a drone, it expects to be able to find the `/etc/starling/vehicle.config` file. This file
+contains some information that MAVROS needs to be able to communicate with the flight controller. An example
+`vehicle.config` file is included below.
+
+If the container is started as part of a Kubernetes StatefulSet deployment, the setup script will attemp# Mavros /ROS12 Bridge Container
+
+To run:
+```bash
+make build # Builds docker container
+make run # Runs docker container
+```
+
+Default action of launching the container (make run) will be to run `launch.sh` which will source mavros_setup.sh, and run the launch file.
+
+## Launch file
+
+The launch file `mavros_bridge.launch` runs the following
+Running the container will run 2 ros nodes in parallel:
+1. A Mavros node listening on fcu_url, targeting drone with sysid tgt_system
+   - `roslaunch mavros apm.launch fcu_url:=${MAVROS_FCU_URL} tgt_system:=${MAVROS_TGT_SYSTEM}`
+2. A ros1 bridge node with option to forward all mavros topics to ros2
+   - `ros2 run ros1_bridge dynamic_bridge --bridge-all-1to2-topics`
```

Comment:
Bridge references should be removed I assume? Or are we still running a bridge?

Reply:
Maybe include references to estop too? Unless separated out as suggested below.

Reply:
Removed and added references